src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/formRenderer.scss

```diff
@@ -1,12 +1,12 @@
 @import 'sulu-admin-bundle/containers/Application/colors.scss';
 
-$cardBackgroundColor: $white;
-$cardBorderColor: $silver;
+$backgroundColor: $white;
+$borderColor: $silver;
 $labelColor: $scorpion;
```

Comment:
Actually I meant that we should prefix all of these variables, just not with card :see_no_evil: It should be something like `fomRendererBackgroundColor` and so on. This might make it easier to override styles later on.